src/main/java/com/ea/restaurant/exceptions/EntityNotFoundException.java

```diff
@@ -0,0 +1,7 @@
+package com.ea.restaurant.exceptions;
+
+public class EntityNotFoundException extends RuntimeException {
```

Comment:
Either rename the class name to be more specific, or use a more generic message.

Reply:
Deleted. It will be implemented later.

Reply:
Done